1.7/administration/installing/custom/configuration-parameters.md

```diff
@@ -57,27 +57,21 @@ This parameter specifies the type of storage backend to use for Exhibitor. You c
        **Important:** With `shared_filesystem`, all masters must must have the NFS volume mounted and `exhibitor_fs_config_dir` must be inside of it. If any of your servers are missing the mount, the DC/OS cluster will not start.
 
 ### <a name="master"></a>master_discovery
-This required parameter specifies the Mesos master discovery method. The available options are `static` or `vrrp`.
+This required parameter specifies the Mesos master discovery method. The available options are `static` or `master_http_loadbalancer`.
 
 *  `master_discovery: static`
 This option specifies that Mesos agents are used to discover the masters by giving each agent a static list of master IPs. The masters must not change IP addresses, and if a master is replaced, the new master must take the old master's IP address. If you specify `static`, you must also specify this parameter:
 
     *  **master_list**
        This required parameter specifies a list of your static master IP addresses as a YAML nested series (`-`).
 
-*  `master_discovery: vrrp`
-This option specifies that Keepalived with a VIP is used to discover the master. You are required to maintain this VIP infrastructure. If you specify `vrrp`, you must also specify these parameters:
-
-    *  **keepalived_router_id**
-       This parameter specifies the virtual router ID of the Keepalived cluster. You must use the same virtual router ID across your cluster.
-    *  **keepalived_interface**
-       This parameter specifies the interface that Keepalived uses.
-    *  **keepalived_pass**
-       If you've set your `auth_type` to `PASS`, this parameter specifies the password that you set for `auth_pass` in your Keepalived configuration file.
-    *  **keepalived_virtual_ipaddress**
-       This parameter specifies the VIP in use by your Keepalived cluster.
+*  `master_discovery: master_http_loadbalancer`
+This option specifies that the set of masters has an HTTP Load balancer in front of them. The slaves will know the address of the load balancer, and can use the load balancer to access exhibitor on the masters, in order to get the full list of Master IPs. If you specify `master_http_load_balancer`, you must also specify these parameters:
+    * **exhibitor_address**
+      This required parameter specifies a (preferrably IP address), that the load balancer in front of the masters live at. The load balancer must forward traffic on port 8181 to port 8181 on the masters,
+      doing something like round robin to pick a master, and ideally avoiding machines which don't respond to requests on port 5050, to ensure Mesos Master is up. A sample load balancer for AWS can be seen in the [DC/OS CloudFormation templates](https://github.com/dcos/dcos/blob/1.7-open/gen/aws/templates/cloudformation.json#L620)
```

Comment:
Suggest: "This required parameter specifies an address (preferrably an IP address), where the load balancer in front of the masters can be found. The load balancer must forward traffic on port 8181 to port 8181 on the masters. It should employ a method like round robin to pick a master and should avoid machines that do not respond to requests on port 5050 in order to ensure the Mesos master is up."